createdisk.sh

```diff
@@ -156,9 +176,15 @@ ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo systemctl stop kubelet
 ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo crictl stopp $(sudo crictl pods -q)'
 ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo crictl rmp $(sudo crictl pods -q)'
 
-# Get rhcos ostree Hash ID
+# Get the rhcos ostree Hash ID
 ostree_hash=$(${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'cat /proc/cmdline | grep -oP "(?<=rhcos-).*(?=/vmlinuz)"')
 
+# Get the rhcos kernel release
+kernel_release=$(${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'uname -r')
+
+# SCP the vmlinuz/initramfs from VM to Host in provided folder.
+${SCP} core@api.${CRC_VM_NAME}.${BASE_DOMAIN}:/boot/ostree/rhcos-${ostree_hash}/* $1
+
```

Comment:
vmlinuz/initramfs are the only 2 files contained in `/boot/ostree/rhcos-${ostree_hash}/`? or do we need to be more specific in what we copy?

Reply:
yes it is only 2 files which are present there.